cmake/Modules/LocalAom.cmake

```diff
@@ -89,13 +89,15 @@ else()
     endif()
 
     if(AVIF_CODEC_AVM)
+        # TODO(vrabaud) remove the PATCH_COMMAND once upstream is fixed.
```

Comment:
Optional: It would be good to link to a merge request for the AVM upstream.

Reply:
No need for a patch anymore as I disable tensorflow. I followed-up with and filed an issue: https://gitlab.com/AOMediaCodec/avm/-/issues/504